doc/modules/feature_extraction.rst

```diff
@@ -490,6 +497,10 @@ class::
 Again please see the :ref:`reference documentation
 <text_feature_extraction_ref>` for the details on all the parameters.
 
+|details-start|
+Example
```

Comment:
Maybe we can be a bit more explicit here:

```suggestion
Numeric example of a tf-idf matrix
```

Reply:
I've done it, thanks for the suggestion! Kindly check.

Reply:
Good to go? @ArturoAmorQ